camel-idea-plugin/src/main/java/com/github/cameltooling/idea/completion/header/CamelHeaderNameCompletion.java

```diff
@@ -39,7 +41,8 @@
 import org.jetbrains.annotations.Nullable;
 
 /**
- * Base class of all supported {@link CompletionProvider}s that give the name of potential headers that can be proposed
+ * Base class of all supported {@link CompletionProvider}s that give the name of
+ * potential headers that can be proposed
```

Comment:
Please remove your commit "Replace Conditional with polymorphism" as it doesn't bring much but reformatting and it adds a new class for each icon which seems to be overkill